coalib/__init__.py

```diff
@@ -6,10 +6,69 @@
 
 
 import sys
+from os import chdir, getcwd
 from os.path import join, dirname
+from subprocess import check_output, CalledProcessError
 
 
-VERSION_FILE = join(dirname(__file__), 'VERSION')
+COALIB_DIR = dirname(__file__)
+
+VERSION_FILE = join(COALIB_DIR, 'VERSION')
+
+COMMIT_FILE = join(COALIB_DIR, 'COMMIT')
+
+
+def get_current_commit(git_repo_dir):
+    """
+    :param git_repo_dir:    string containing the directory to check the
+                            current commit in.
+    :return:                string containing the current commit sha if
+                            `git_repo_dir`is initialized as a git repo
+                            otherwise an empty string.
```

Comment:
Start sentences with upper-case + Alignment (see below for recommendation on how to format docs)